src/components/FeatureViews/AdminPortal/AdminActionCenter.js

```diff
@@ -73,38 +75,41 @@ const AdminActionCenter = () => {
 	return (
 		<Grid className="admin-actions-wrapper" container spacing={2}>
 			<Grid item>
-				<Button
+				<ResponsiveButton
 					aria-controls="simple-menu"
 					aria-haspopup="true"
 					className={`button ${tabState.user && "active"}`}
 					onClick={handleClick(setUserAnchor)}
+					variant='outlined'
 				>
-					<NewInstructor className="admin-action-icon" />
+					{/* <NewInstructor className="admin-action-icon" /> */}
```

Comment:
Is this line not needed? If so get rid of it. also lines 97 and 110 have similar comments